packages/dart_frog_lint/example/main.dart

```diff
@@ -0,0 +1,7 @@
+import 'dart:io';
+
+import 'package:dart_frog/dart_frog.dart';
+
+Future<HttpServer> run(Handler handler, InternetAddress ip, int port) {
+  throw UnimplementedError();
+}
```

Comment:
Currently we are not testing that the lint rules `dart_frog_entrypoint` works as expected. In other words, if the `DartFrogEntrypoint` implementation is altered to have an incorrect behaviour or no behaviour at all, the CI will not complain about it.

We might need another testing strategy for this lint rule? Since we can't really have two `main.dart` under the same directory?

In [here](https://github.com/felangel/mocktail/blob/b7033740570b4b6fc3fa7d65783f5138d0069645/packages/mocktail_lint/test/rules/prefer_private_mocks_test.dart) we're using an actual test alongside the `example/` testing (note that is very WIP) but I would still would like the `example/` to include examples of the lint rule failing and being catched by `expect_lint`.

Reply:
What do you think about changing `filesToAnalyze` to:

```dart
  @override
  List<String> get filesToAnalyze => ['main.dart', 'dart_frog_entrypoint/**.dart'];
```

Then we could have example/dart_frog_entrypoint/whatever.dart with expect_lint in it?